UnitedIncome/components
# Cabinet Linkint

### Description
> Please provide a brief description of your changes below.

Adds cabinet styling for `sup` > `a` links. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Click on the default cabinet story and observe the `1` link. Once clicked it will scroll down to the footnotes.

### Comments
> Any additional comments that will aid with the review process.

Surprised this works! Happy days.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
